src/keys-builder/utils/copy-without-missing-keys.ts

```diff
@@ -0,0 +1,22 @@
+import { isObject } from '../../utils/validators.utils';
+import { Translation } from '../../types';
+
+export function copyWithoutMissingKeys(
+  currentTranslation: Translation,
+  extractedTranslation: Translation
+): Translation {
+  const cleanedTranslation: Translation = {};
+
+  for (const key in currentTranslation) {
+    if (extractedTranslation[key] === undefined) continue;
```

Comment:
Please see [this comment](https://github.com/ngneat/transloco-keys-manager/pull/89#discussion_r1050846882).